### PR TITLE
Cocoapods support

### PR DESCRIPTION
**Pull request by izqui**
I prepared the `podspec`, but it still needs some work.

Instructions for merging:

* Complete .podspec information, some of the fields such us authors or socialmedia where left blank.
* `pod lib lint`
* Tag release and push to github
* `pod spec lint`
* Push spec to the Cocoapods trunk

### CODE REVIEW

File: SwiftSockets.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                       A simple GCD based socket wrapper for Swift.
+DESC
+  s.homepage         = "https://github.com/AlwaysRightInstitute/SwiftSockets"
+
+  s.license          = 'MIT'
+  s.author           = { "Helge Heß" => "email@email.com" }
+  s.source           = { :git => "https://github.com/AlwaysRightInstitute/SwiftSockets.git", :tag => s.version.to_s }
+  s.social_media_url = 'http://twitter.com/'
+
+  s.platform     = :osx, '10.10'
```
Comment (alexperezpaya) [inline SwiftSockets.podspec]: What about iOS Support? @izqui

**Comment by izqui** [comment]
I'm not sure if that would work since it interfaces directly with Darwin, but I'm sure @helje5 know better about this

**Comment by helje5** [comment]
What wouldn't work?

**Comment by Danappelxx** [comment]
@helje5 he's saying that he set it to OSX only because he's not sure if Darwin is available on iOS. Can you confirm/deny that?

**Comment by helje5** [comment]
iOS is Darwin as well, that is not an issue. The lib itself should work fine on iOS and OSX (but I never checked whether the Xcode project is setup for that [and no idea what would be required]).

The sample apps on the other hand use Cocoa and obviously won't work on iOS.

**Comment by izqui** [comment]
It is now a mac and osx pod ready to be pushed in to the cocoapods trunk

**Pull request merged**